Documentation/ApiOverview/GlobalValues/Typo3ConfVars/BE.rst

```diff
@@ -668,6 +682,29 @@ $GLOBALS['TYPO3_CONF_VARS']['BE']['defaultPageTSconfig']
 
    Contains the default page TSconfig.
 
+   Never set this configuration variable directly. Use the following methods instead:
+
+   .. versionadded:: 12.0
+      TSconfig stored in a file :file:`EXT:my_sitepackage/Configuration/page.tsconfig`
+      will be automatically added to :php:`$TYPO3_CONF_VARS[SYS][defaultPageTSconfig]`.
+
+   Page TSconfig stored in a files from all extensions called for example
+   :file:`EXT:my_sitepackage/Configuration/page.tsconfig` will automatically be
+   loaded into :php:`$TYPO3_CONF_VARS[SYS][defaultPageTSconfig]`. This is done
```

Comment:
thats not correct as such: the stuff loaded from page.tsconfig is handled, remaining $TYPO3_CONF_VARS[SYS][defaultPageTSconfig] is then appended during bootstrap. The basic idea is to get rid of both $TYPO3_CONF_VARS[SYS][defaultPageTSconfig] and ExtensionManagementUtility::addPageTSConfig() since they suck and create a mess during runtime  (but maybe get a better wording on that last part ;)

Reply:
Ok, seems like I missunderstood this. Changed the text.